source/compatibility.tex

```diff
@@ -1597,6 +1597,18 @@
           // in \tcode{A<long>(long)}.
 \end{codeblock}
 
+\rSec2[diff.cpp14.constraints]{Clause \ref{constraints}: Constraints on programs }
+
+\ref{namespace.future}
+\change New reserved namespaces
+\rationale Reserve namespaces for future revisions of the standard library that
+might otherwise be incompatible with existing programs.
+\effect
+The global namespaces \tcode{std} followed by an arbitrary sequence of digits
```

Comment:
"decimal digits"? The standard defines binary-, octal-, decimal-, and hexadecimal-digit in [lex.icon].